uploadNeuroDB/NeuroDB/FileDecompress.pm

```diff
@@ -1,4 +1,32 @@
 package NeuroDB::FileDecompress;
+
+=pod
+
+=head1 NAME
+
+NeuroDB::FileDecompress -- Provides an interface to the file decompression of
+ LORIS
```

Comment:
maybe LORIS-MRI instead of LORIS? I think LORIS has its own decompression class.